### PR TITLE
Feature/widget tests

### PR DESCRIPTION
**Pull request by silverkate**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -66,12 +66,19 @@ dependencies:
   tab_indicator_styler: ^2.0.0
   extended_nested_scroll_view: ^6.1.2
 
+  #testing
```
Comment (VSydorko) [inline pubspec.yaml:69]: @silverkate move it above #widgets section

File: test/widget_test/widget/app_drawer/app_drawer_test.dart [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      final bloc = MockAuthBloc();
+
+      /// Get the widget.
+      await $.pumpWidget(
+        EasyLocalization(
+          path: CodegenLoader.path,
+          supportedLocales: CodegenLoader.supportedLocales,
+          fallbackLocale: CodegenLoader.supportedLocales.last,
+          assetLoader: const CodegenLoader(),
+          child: Builder(
```
Comment (VSydorko) [inline test/widget_test/widget/app_drawer/app_drawer_test.dart:35]: @silverkate Can we remove Builder?

File: test/widget_test/widget/text_input_form_builder/text_input_form_builder_test.dart [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:stx_flutter_form_bloc/stx_flutter_form_bloc.dart';
+
+Future<void> main() async {
+  /// Init EasyLocalization.
+  SharedPreferences.setMockInitialValues({});
+  await EasyLocalization.ensureInitialized();
+
+  /// Call ensureInitialized, because main is async.
+  TestWidgetsFlutterBinding.ensureInitialized();
```
Comment (VSydorko) [inline test/widget_test/widget/text_input_form_builder/text_input_form_builder_test.dart:15]: @silverkate Can we remove it?

File: ios/Podfile [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '11.0'
+platform :ios, '13.0'
```
[the remark this answers is not in the document]
Comment (silverkate) [reply ios/Podfile:2]: it is required to run 'pod install' after including patrol

File: test/widget_test/widget/app_drawer/app_drawer_test.dart (hunk 2 above)
Comment (VSydorko) [reply test/widget_test/widget/app_drawer/app_drawer_test.dart:37]: I see

File: test/widget_test/widget/text_input_form_builder/text_input_form_builder_test.dart [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      /// Mock bloc interactions.
+      final bloc = TextFieldBloc(
+        required: true,
+        rules: {ValidationType.onChange},
+        customValidators: {FieldBlocValidators.passwordMin6Chars},
+      );
+
+      /// Get the widget.
+      await $.pumpWidget(
+        EasyLocalization(
```
Comment (VSydorko) [inline test/widget_test/widget/text_input_form_builder/text_input_form_builder_test.dart:28]: Why did you wrap TextInputFormBuilder with EasyLocalization?

File: ios/Podfile (hunk 4 above)
[the remark this answers is not in the document]
Comment (VSydorko) [reply ios/Podfile:2]: bad thing. we have to come up with some ideas how to set this setting in the new project automatically.

[the remark this answers is not in the document]
Comment (silverkate) [reply ios/Podfile]: Another possible variant to resolve the issue is to make some changes to `ios/Runner.xcodeproj/project.pbxproj` file. I've commited the solution

[the remark this answers is not in the document]
Comment (VSydorko) [reply ios/Podfile]: @silverkate we can't have the iOS folder in the guildelines brick. Please revert your changes
Probably we will create a separate brick for tests.

[the remark this answers is not in the document]
Comment (VSydorko) [reply ios/Podfile]: at least uncomment `platform :ios, '13.0'`

**Pull request merged**